fishnet.py

```diff
@@ -1714,6 +1789,101 @@ def cmd_run(args):
         print("Ignored deprecated option --threads. Did you mean --cores?")
         print()
 
+    return cores, threads, instances, memory, user_backlog, system_backlog
+
+
+def cmd_benchmark(args):
+    conf = load_conf(args)
+    cores, threads, instances, memory, _, _ = display_config(args, conf)
+
+    buckets = [0] * instances
+    for i in range(0, cores):
+        buckets[i % instances] += 1
+
+    instance_memory = memory // instances
+    workers = [BenchmarkWorker(conf, bucket, instance_memory) for bucket in buckets]
+
+    logging.info("Starting %d workers" % len(workers))
+
+    # Start all threads
+    for i, worker in enumerate(workers):
+        worker.set_name("><> %d" % (i + 1))
+        worker.setDaemon(True)
+        worker.start()
+
+    # Note: Waiting with timeout on worker.running would be nicer.
```

Comment:
Agreed. Let's have this.

Alternatively: Why do we have to wait for this in the first place? If it's about calling `stop_soon()` at the right moment, maybe the worker could be further refactored to add a better mechanism to run only one job.

Reply:
Done, it's a little funky because the first worker only get's 1 second and later workers get more time.